bundler/lib/bundler/definition.rb

```diff
@@ -71,6 +71,7 @@ def initialize(lockfile, dependencies, sources, unlock, ruby_version = nil, opti
       else
         @unlocking_all = false
         @unlocking_bundler = unlock.delete(:bundler)
+        @validate_dependencies = unlock.delete(:validate_dependencies)
```

Comment:
Like `conservative`, this is only used in initialize, so we could use a local variable instead.

Also, again regarding naming and following the rationale of the previous comment, I think maybe I'd use the negative value here and name it `skip_validating_locally_installed_specifications`.

Reply:
Indeed!

I'm not following what you were saying about the "negative", but I tried to give it an appropriate name.

Reply:
I just meant saving `!unlock.delete(:validate_dependencies)` (hence the "skip_" name since it has the opposite meaning), so that we can then check a positive condition in `LazySpecifcation` like

```ruby
      if @materialization_options[:skip_validating_locally_installed_dependencies] && spec.is_a?(StubSpecification)
```

But nevermind, I think it's fine either way.